pkg/gooffensive/process_windows.go

```diff
@@ -0,0 +1,59 @@
+package gooffensive
+
+import (
+	"fmt"
+	"syscall"
+	"unsafe"
+)
+
+const (
+	// TH32CS_SNAPPROCESS is a Windows constant used with Windows API calls
+	TH32CS_SNAPPROCESS = 0x00000002
```

Comment:
`TH32CS_SNAPPROCESS` is already in the `syscall` package, so I think we can just use `syscall.TH32CS_SNAPPROCESS`.